opam

```diff
@@ -19,7 +19,7 @@ build: [
 install: [make "install" "INSTALL_DIR=%{prefix}%"]
 remove: [make "uninstall" "INSTALL_DIR=%{prefix}%"]
 depends: [
-  "ocaml" {>= "4.02"}
+  "ocaml" {>= "4.12"}
```

Comment:
@andyli do we still have to stick to 4.02?

Reply:
Ubuntu 16.04 reached its [End of Standard Support](https://wiki.ubuntu.com/Releases) earlier this year. We can bump the min ocaml version to 4.05.0, which is provided by Ubuntu 18.04.